Tests/DatadogTests/Datadog/RUM/WebView/WKUserContentController+DatadogTests.swift

```diff
@@ -61,6 +67,35 @@ class WKUserContentController_DatadogTests: XCTestCase {
         XCTAssertEqual(sanitization.warningMessage, "The allowed WebView host configured for Datadog SDK is not valid")
     }
 
+    func testItAddsUserScriptAndMessageHandlerMultipleTimes() throws {
```

Comment:
The implementation of this test is fine, but I don't understand its name - sounds like we expect the controller to add handler multiple times.
```suggestion
    func testWhenAddingMessageHandlerMultipleTimes_itIgnoresExtraOnesAndPrintsWarning() throws {
```